components/button/index.css

```diff
@@ -106,6 +104,258 @@ governing permissions and limitations under the License.
+    --spectrum-button-background-color-down: var(--spectrum-transparent-white-900);
+    --spectrum-button-background-color-focus: var(--spectrum-transparent-white-900);
+
+    --spectrum-button-content-color-default: var(--spectrum-black);
+    --spectrum-button-content-color-hover: var(--spectrum-black);
+    --spectrum-button-content-color-down: var(--spectrum-black);
+    --spectrum-button-content-color-focus: var(--spectrum-black);
+
+    --spectrum-button-focus-indicator-color: var(--spectrum-static-white-focus-indicator-color);
+
+    --spectrum-button-background-color-disabled: var(--spectrum-disabled-static-white-background-color);
+    --spectrum-button-border-color-disabled: transparent;
+    --spectrum-button-content-color-disabled: var(--spectrum-disabled-static-white-content-color);
+
+    &.spectrum-Button--outline {
+      --spectrum-button-background-color-default: var(--spectrum-transparent-white-25);
+      --spectrum-button-background-color-hover: var(--spectrum-transparent-white-100);
+      --spectrum-button-background-color-down: var(--spectrum-transparent-white-100);
+      --spectrum-button-background-color-focus: var(--spectrum-transparent-white-100);
+
+      --spectrum-button-border-color-default: var(--spectrum-transparent-white-800);
+      --spectrum-button-border-color-hover: var(--spectrum-transparent-white-900);
+      --spectrum-button-border-color-down: var(--spectrum-transparent-white-900);
+      --spectrum-button-border-color-focus: var(--spectrum-transparent-white-900);
+
+      --spectrum-button-content-color-default: var(--spectrum-transparent-white-800);
+      --spectrum-button-content-color-hover: var(--spectrum-transparent-white-900);
+      --spectrum-button-content-color-down: var(--spectrum-transparent-white-900);
+      --spectrum-button-content-color-focus: var(--spectrum-transparent-white-900);
+
+      --spectrum-button-background-color-disabled: var(--spectrum-transparent-white-25);
+      --spectrum-button-border-color-disabled: var(--spectrum-disabled-static-white-border-color);
+      --spectrum-button-content-color-disabled: var(--spectrum-disabled-static-white-content-color);
+    }
+
+    &.spectrum-Button--secondary {
+      --spectrum-button-background-color-default: var(--spectrum-transparent-white-100);
+      --spectrum-button-background-color-hover: var(--spectrum-transparent-white-200);
+      --spectrum-button-background-color-down: var(--spectrum-transparent-white-200);
+      --spectrum-button-background-color-focus: var(--spectrum-transparent-white-200);
+  
+      --spectrum-button-content-color-default: var(--spectrum-transparent-white-800);
+      --spectrum-button-content-color-hover: var(--spectrum-transparent-white-900);
+      --spectrum-button-content-color-down: var(--spectrum-transparent-white-900);
+      --spectrum-button-content-color-focus: var(--spectrum-transparent-white-900);
+  
+      &.spectrum-Button--outline {
```

Comment:
I think we might be missing the background changes for this secondary static white with outline

Reply:
Good catch 👍 I've pushed up a commit that fixes this.